Donation-platform/src/main/webapp/personalizationScript.js

```diff
@@ -91,6 +82,47 @@ function updatePersonalizedCardsOnPage(charities) {
     cards.innerHTML += toAdd;
 }
 
+/* Converts the list of sortable-compitable tag values (which require underscores) 
+   into a list of the top 3 tags with their proper names.
+
+   Example: tagOrder = ["li_health", "li_hunger", "li_education", "li_children", 
```

Comment:
give an input output example

Reply:
I made it more clear that 'tagOrder' and 'tags' are respectively the input and output by replacing their names with input/output in the comment.